include/zephyr/drivers/dma.h

```diff
@@ -66,6 +66,14 @@ enum dma_channel_filter {
 	DMA_CHANNEL_PERIODIC, /* can be triggered by periodic sources */
 };
 
+/* DMA attributes */
+enum dma_attribute_type {
+	DMA_ATTR_BUFFER_ALIGNMENT,
+	DMA_ATTR_COPY_ALIGNMENT,
+	DMA_ATTR_BUFFER_ADDRESS_ALIGNMENT,
+	DMA_ATTR_BUFFER_PERIOD_COUNT,
```

Comment:
Is "period" a generic DMA term? This has established meaning in audio domain, but not sure if it's correct to use in generic DMA driver interface.

Reply:
Thats right, period should be app (sof) specific. I can see we already have DW_DMA_BUFFER_PERIOD_COUNT defined in zephyr dma_dw_common.h without usage, probably should be removed